### ISSUE TITLE
something wrong set WCJS_RUNTIME_VERSION

### ISSUE THREAD

**Issue by s-zk** [jaruba/node-vlcPlayer-demo#11]
Dear jaruba,

Thank you for such an amazing nodejs package. I have encountered some trouble when installing wcjs-prebuilt. As the readme.md said, I need to set 

```
set WCJS_RUNTIME=electron
set WCJS_RUNTIME_VERSION=0.37.6
```

then installation failed with following:

```
WebChimera.js_electron_0.33.9_ia32_win.zip  doesn't include VLC
WebChimera.js_electron_0.33.9_x64_osx.zip  doesn't include VLC
WebChimera.js_electron_0.33.9_x64_win.zip  doesn't include VLC
WebChimera.js_electron_0.37.6_ia32_win.zip  doesn't include VLC
WebChimera.js_electron_0.37.6_x64_osx.zip  doesn't include VLC
WebChimera.js_electron_0.37.6_x64_win.zip  doesn't include VLC
WebChimera.js_electron_1.2.0_ia32_win.zip  doesn't include VLC
WebChimera.js_electron_1.2.0_x64_osx.zip  doesn't include VLC
WebChimera.js_electron_1.2.0_x64_win.zip  doesn't include VLC
WebChimera.js_nw_0.15.0_ia32_win.zip  doesn't include VLC
WebChimera.js_nw_0.15.0_x64_osx.zip  doesn't include VLC
WebChimera.js_nw_0.15.0_x64_win.zip  doesn't include VLC
WebChimera.js_v0.2.5_electron_v0.33.9_VLC_v2.2.3_ia32_win.zip  not matching environment
WebChimera.js_v0.2.5_electron_v0.33.9_VLC_v2.2.3_x64_osx.tar.gz  not matching environment
WebChimera.js_v0.2.5_electron_v0.33.9_VLC_v2.2.3_x64_win.zip  not matching environment
WebChimera.js_v0.2.5_electron_v0.37.6_VLC_v2.2.3_ia32_win.zip  not matching environment
WebChimera.js_v0.2.5_electron_v0.37.6_VLC_v2.2.3_x64_osx.tar.gz  not matching environment
WebChimera.js_v0.2.5_electron_v0.37.6_VLC_v2.2.3_x64_win.zip  not matching environment
WebChimera.js_v0.2.5_electron_v1.2.0_VLC_v2.2.3_ia32_win.zip  not matching environment
WebChimera.js_v0.2.5_electron_v1.2.0_VLC_v2.2.3_x64_osx.tar.gz  not matching environment
WebChimera.js_v0.2.5_electron_v1.2.0_VLC_v2.2.3_x64_win.zip  not matching environment
WebChimera.js_v0.2.5_nw_v0.15.0_VLC_v2.2.3_ia32_win.zip  not matching environment
WebChimera.js_v0.2.5_nw_v0.15.0_VLC_v2.2.3_x64_osx.tar.gz  not matching environment
WebChimera.js_v0.2.5_nw_v0.15.0_VLC_v2.2.3_x64_win.zip  not matching environment

No WebChimera release found matching your environment
npm ERR! Windows_NT 10.0.10586
npm ERR! argv "D:\\Program Files\\nodejs\\node.exe" "D:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "wcjs-prebuilt"
npm ERR! node v5.10.1
npm ERR! npm  v3.8.3
npm ERR! code ELIFECYCLE

npm ERR! wcjs-prebuilt@3.0.0 postinstall: `node install.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the wcjs-prebuilt@3.0.0 postinstall script 'node install.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the wcjs-prebuilt package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node install.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs wcjs-prebuilt
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls wcjs-prebuilt
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     D:\test\node-vlcPlayer-demo\npm-debug.log

```

I am on Windows 10, 64-bit, node v 5.10.1 and npm 3.8.3. 

Thank you in advance.

**Comment by s-zk** [comment]
I solved it by using `set WCJS_RUNTIME_VERSION=v0.37.6`. Hope it helps.

**Comment by jaruba** [comment]
I fixed the readme examples on all the repos, thanks for finding it! :)
